### PR TITLE
Added behavior linking

### PR DESCRIPTION
**Pull request by EmosewaMC**
Added behavior referencing for LUdocs to directly generate the links to explorer.lu

tested on mac and links generate correctly.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -163,6 +163,7 @@ def role(role, rawtext, text, lineno, inliner,
         return [node], []
     return role
 
+behaviors_role = explorer_role('Behaviors ', 'skills/behaviors/%d')
```
Comment (Xiphoseer) [inline docs/conf.py:166]:
```suggestion
behaviors_role = explorer_role('Behavior ', 'skills/behaviors/%d')
```
Didn't notice that with the other additions, but this should be singular to produce sth like `Behavior 2345` as a link text

Comment (EmosewaMC) [reply docs/conf.py:166]: Do you want me to make that change to the other roles that are plural and change them all to be singular?

Comment (Xiphoseer) [reply docs/conf.py:166]: Sure, no need for separate PRs

**Comment by EmosewaMC** [comment]
removed the plurals and replaced their names with singulars.

**Pull request merged**